docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -806,4 +806,163 @@ The following is performed on ArduCopter version v4.5.7
 
 .. image:: ../_static/manual_images/dronecan/ardupilot_dna_success.PNG
 
-6. Now, if you reboot your module, and check the DroneCAN configuration window again, you'll find that ArduPilot allocates the same ID again
\ No newline at end of file
+6. Now, if you reboot your module, and check the DroneCAN configuration window again, you'll find that ArduPilot allocates the same ID again
+
+*************************************
+Calculating DroneCAN Utilization
```

Comment:
Maybe title this "Calculating DroneCAN Bus Utilization", adding the bus bit, since the first sentence after this calls it "calculating DroneCAN bus utilization"